packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -27,5 +27,6 @@ AdminUI.defaultProps = {
     catchAll: NotFound,
     loading: LoadingPage,
     loginPage: Login,
+    loginCallbackPage: LoginCallback,
```

Comment:
I think you must do the same in the `react-admin` package

Reply:
Nope